packages/amplify-console-integration/package.json

```diff
@@ -59,7 +59,15 @@
     ],
     "reporters": [
       "default",
-      "jest-junit"
+      "jest-junit",
```

Comment:
can we name this package as `amplify-console-integration-tests` so its clear this is a test package

Reply:
check